### PR TITLE
Added usage examples for the draw_line function

### PR DESCRIPTION
**Pull request by SimonRhook**
# Overview

Added usage examples for the draw_line function including C\# OOP and Top Level, C++ and python 

# Files Included

Splashkit Function: `function_name`
- [x] Title and explanation (.txt)
- [x] C++ code
- [x] C# code (top-level statements)
- [x] C# code (Object-Oriented Programming)
- [x] Python code
- [x] Screenshot

# Usage Example Checks

- [x] Simple, clear demonstration of the function
- [x] Code uses Splashkit functions
- [x] Tested in Chrome and Firefox

### CODE REVIEW

**Review by Jinx-means-jinx — COMMENTED** [review]
Check the code for oop and running with no error
image

Check the code for top level and running with no error
image

Check the code for C++ and running with no error
image

Check the code for python and running with no error
image

**Comment by Jinx-means-jinx** [comment]
Great work!

File: public/usage-examples/graphics/draw_line/draw_line-1-simple-oop.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create Window
+            SplashKit.OpenWindow("Draw Line", 600, 600);
+            SplashKit.ClearScreen(SplashKit.ColorBlack());
+
+
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line/draw_line-1-simple-oop.cs:13]: remove this line

File: public/usage-examples/graphics/draw_line/draw_line-1-simple-top-level.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+using SplashKitSDK;
+using static SplashKitSDK.SplashKit;
+
+
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line/draw_line-1-simple-top-level.cs:4]: and this one

File: public/usage-examples/graphics/draw_line/draw_line-1-simple-top-level.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+using SplashKitSDK;
+using static SplashKitSDK.SplashKit;
+
+
+// Create Window
+OpenWindow("Draw Line", 600, 600);
+ClearScreen(ColorBlack());
+
+
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line/draw_line-1-simple-top-level.cs:9]: and this one

File: public/usage-examples/graphics/draw_line/draw_line-1-simple.cpp [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#include "splashkit.h"
+
+int main()
+{
+    // Create Window
+    open_window("Draw Line", 600, 600);
+    clear_screen(COLOR_BLACK);
+
+
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line/draw_line-1-simple.cpp:9]: and this one

File: public/usage-examples/graphics/draw_line/draw_line-1-simple.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+from splashkit import *
+
+#Create Window
+open_window("Draw Line", 600, 600)
+clear_screen(color_black())
+
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_line/draw_line-1-simple.py:6]: and this one too

**Review by breezy-codes — CHANGES_REQUESTED** [review]
I've reviewed this PR, all code runs well and free of errors.
I just have a few formatting requests, it also might be good to be a smidge more descriptive in the txt file, like what DO the lines form?

**Review by breezy-codes — APPROVED** [review]
I've reviewed this PR, all code files run correctly and are free of any errors. Code comments are consistent and so is the code.

I approve of this PR.


- [X] All required files are present.
  - [X] Title and explanation (.txt)
  - [X] C++ code
  - [X] C# code (top-level statements)
  - [X] C# code (Object-Oriented Programming)
  - [X] Python code
- [X] Code correctly uses Splashkit functions.
- [X] Code clearly demonstrates the function.
- [X] All versions maintain the same structure and comments.
- [X] C++ code ran correctly.
- [X] C# top level code ran correctly.
- [X] C# OOP code ran correctly.
- [X] Python code ran correctly.

**Review by omckeon — APPROVED** [review]

**Pull request merged**